Cargo.toml

```diff
@@ -21,6 +21,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 [dependencies]
 arbitrary = { default-features = false, optional = true, version = "1.0" }
 arrayvec = { default-features = false, version = "0.7" }
+bitcode = { version = "0", default-features = false, features = ["derive"], optional = true }
```

Comment:
[Add bitcode to libraries without specifying the major version so binary crates can pick the version.](https://github.com/SoftbearStudios/bitcode/blob/15fe3aa8393276c0312896478e2b81d4e28267da/README.md?plain=1#L40)